source/Ubuntu-B.ufo/fontinfo.plist

```diff
@@ -75,16 +75,16 @@
 		<integer>-189</integer>
 		<key>openTypeHheaLineGap</key>
 		<integer>28</integer>
-		<key>openTypeNameDescription</key>
-		<string>The Ubuntu Font Family are libre fonts funded by Canonical Ltd on behalf of the Ubuntu project. The font design work and technical implementation is being undertaken by Dalton Maag. The typeface is sans-serif, uses OpenType features and is manually hinted for clarity on desktop and mobile computing screens.
-
-The scope of the Ubuntu Font Family includes all the languages used by the various Ubuntu users around the world in tune with Ubuntu's philosophy which states that every user should be able to use their software in the language of their choice. The project is ongoing, and we expect the family will be extended to cover many written languages in the coming years.</string>
 		<key>openTypeNameDesigner</key>
-		<string>Dalton Maag Ltd</string>
+		<string>Dalton Maag Limited</string>
```

Comment:
Might be better to fix this upstream by teaching the checker that "Ltd" is an abbreviation of "Limited".